src/main/java/com/github/cameltooling/lsp/internal/completion/CamelComponentSchemesCompletionsFuture.java

```diff
@@ -49,5 +58,21 @@ public CamelComponentSchemesCompletionsFuture(CamelUriElementInstance uriElement
 				})
 				.filter(FilterPredicateUtils.matchesCompletionFilter(filterString))
 				.collect(Collectors.toList());
+		
+		if (ReferencesProcessor.isDirectComponentKind(uriElement)) {
+			// find all endpoints of same component scheme and add them to the list of completion suggestions
```

Comment:
instead of writing a comment, I think extracting a method would be better for readability

Reply:
sounds good